app/models/concerns/page_manager.rb

```diff
@@ -6,18 +6,37 @@ module PageManager
       rails_admin do
         list do
           field :name
+          field :language
           field :content
         end
-
         edit do
           field :name
+          field :language, :enum do
+            enum do
+              [
+                %w[Français fr],
```

Comment:
Is this array can be populated from the loaded locales?

Reply:
Yeah, working on that actually